src/test/ij.c

```diff
@@ -683,7 +689,7 @@ main( hypre_int argc,
    IS_type = 1;
    debug_flag = 0;
    solver_id = 0;
-   ioutdat = 3;
+   ioutdat = 2;
```

Comment:
Can we include the new iout options (3 and 4 for tagged cases) in the help output?

Reply:
Good idea, done in [d211bda](https://github.com/hypre-space/hypre/pull/1254/commits/d211bda41cc5950c102ceca8a22c7482ea69e792)